gcloud-java-compute/src/test/java/com/google/gcloud/compute/DiskIdTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.compute;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class DiskIdTest {
+
+  private static final String PROJECT = "project";
+  private static final String ZONE = "zone";
+  private static final String NAME = "disk";
+  private static final String URL =
+      "https://www.googleapis.com/compute/v1/projects/project/zones/zone/disks/disk";
+
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testOf() {
+    DiskId diskId = DiskId.of(PROJECT, ZONE, NAME);
+    assertEquals(PROJECT, diskId.project());
+    assertEquals(ZONE, diskId.zone());
+    assertEquals(NAME, diskId.disk());
+    assertEquals(URL, diskId.selfLink());
+    diskId = DiskId.of(ZONE, NAME);
+    assertNull(diskId.project());
+    assertEquals(ZONE, diskId.zone());
+    assertEquals(NAME, diskId.disk());
+    diskId = DiskId.of(ZoneId.of(ZONE), NAME);
+    assertNull(diskId.project());
+    assertEquals(ZONE, diskId.zone());
+    assertEquals(NAME, diskId.disk());
+  }
+
+  @Test
+  public void testToAndFromUrl() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/mziccard/gcloud-java/file/2293826559/issues/source?bid=3140968&fileBranchId=3191561#l58)